<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hydraharp_mainWindow</class>
 <widget class="QMainWindow" name="hydraharp_mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>513</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Simple XY Scan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="hydraharp_modules">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>380</height>
       </size>
      </property>
      <property name="title">
       <string>Controls</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="modules_layout">
       <item>
        <layout class="QVBoxLayout" name="control">
         <item>
          <widget class="QGroupBox" name="Connection_groupBox">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Device Connection Parameters</string>
           </property>
           <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
            <item row="3" column="2">
             <widget class="QComboBox" name="Mode_comboBox"/>
            </item>
            <item row="4" column="2">
             <widget class="QComboBox" name="RefSource_comboBox"/>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Mode:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="DeviceIndex_doubleSpinBox">
              <property name="decimals">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="Mode_label">
              <property name="text">
               <string>DeviceIndex:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>RefSource:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QCheckBox" name="connected_checkBox">
              <property name="text">
               <string>connected</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="Aquisition_GroupBox">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Histogram Acquisition Settings</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="6" column="1">
             <widget class="QDoubleSpinBox" name="Resolution_doubleSpinBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>HistogramBins:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Resolution:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="HistogramBins_doubleSpinBox">
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="maximum">
               <double>100000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>StopCount</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QDoubleSpinBox" name="StopCount_doubleSpinBox"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Binnning:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QCheckBox" name="StopOnOverflow_checkBox">
              <property name="text">
               <string>StopOnOverflow</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Tacq:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>StopOnOverOverflow</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="Binning_comboBox"/>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="Tacq_doubleSpinBox">
              <property name="suffix">
               <string> sec</string>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>40000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="sync">
         <item>
          <widget class="QGroupBox" name="Sync_GroupBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>320</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Sync Channel</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="5" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>SyncPeriod</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="SyncRate_doubleSpinBox"/>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="CFDZeroCrossSync_doubleSpinBox"/>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="SyncDivider_comboBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>SyncDivider</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>CFDZeroCrossSync</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="SyncOffset_doubleSpinBox"/>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="CFDLevelSync_doubleSpinBox"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>CFDLevelSync</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QDoubleSpinBox" name="SyncPeriod_doubleSpinBox"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>SyncRate</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>SyncOffset</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="channel_optimizer_GroupBox">
      <property name="title">
       <string>Channel Optimizer</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="channel_optimizer_layout">
       <item>
        <widget class="QFrame" name="optimizer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>1</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="optimizer_layout">
          <item>
           <layout class="QVBoxLayout" name="B1">
            <item>
             <widget class="QCheckBox" name="run_checkBox">
              <property name="text">
               <string>run</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="SyncRate_visible_checkBox">
              <property name="text">
               <string>show sync channel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QDoubleSpinBox" name="history_len_doubleSpinBox">
              <property name="prefix">
               <string>history len: </string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="avg_len_doubleSpinBox">
              <property name="prefix">
               <string>avg_len: </string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>connected_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Sync_GroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
